app/public/javascripts/views.js

```diff
@@ -46,6 +57,30 @@ allDays.forEach(function(td){
 ///////////////////////////////////////////////
 ////////////////BOOKING VIEW///////////////////
 ///////////////////////////////////////////////
+
+var CreateBookingView = Backbone.View.extend({
+  render: function(){
+    var date = "This day is " + this.model.get("date");
+    var user = " and the user is " + this.model.get("user");
+    var modal = "<div id='bookingModal' class='modal fade' role='dialog'><div class='modal-dialog'>" +
```

Comment:
There is probably some kind of Handlebars method we can use to extract this template out of the file, instead of using a lot of concatenation.